flows/load_image.py

```diff
@@ -76,14 +76,18 @@ def load_image(FILENAME):
 
 	# get image and wcs solution
 	with fits.open(FILENAME, mode='readonly') as hdul:
+
 		hdr = hdul[0].header
 		origin = hdr.get('ORIGIN')
 		telescope = hdr.get('TELESCOP')
 		instrument = hdr.get('INSTRUME')
 
-		image.image = np.asarray(hdul[0].data, dtype='float64')
+		image.image = np.asarray(hdul[0].data, dtype=np.float64)
```

Comment:
Why change this?

Reply:
Part of troubleshooting the endian issues. Np is fine here.